Ajout de la page HTML et des pages js correspondantes pour la création d'articles pour une association.

Gugussss opened the pull request.

Gugussss committed:
creer article final avec js

Gugussss committed:
Augustin creer article js

xzen commented:
Bonsoir merci de mettre des noms de request qui soit utilisable et pas juste avec voilà … bon week-ends

Esscraye committed:
Update creerarticle.html remove Trailing slash on void elements and Stray end tag textarea

suppression du / à la fin de la balise img et suppression du </textarea> après un input.

Esscraye committed:
Update bandeau.js delete "/" at end of <img>

Esscraye committed:
Update creerarticle.js delete </textarea> aefter input

Esscraye committed:
Merge branch 'devel' into devel
